src/solver/variable/economy/all.h

```diff
@@ -168,90 +169,93 @@ typedef                               // Prices
                                    <SpilledEnergy          // Spilled Energy
-                                    <LOLD                  // LOLD
+                                    <LOLD                  // Loss Of Load Duration
                                      <LOLD_CSR<
-                                       LOLP // LOLP
-                                       <LOLP_CSR<AvailableDispatchGen<DispatchableGenMargin<
-                                         DtgMarginCsr // DTG MRG CSR
-                                         <Marge<MaxMrgCsr<NonProportionalCost<
-                                           NonProportionalCostByDispatchablePlant // Startup cost +
-                                                                                  // Fixed cost per
-                                                                                  // thermal plant
-                                                                                  // detail
-                                           <NbOfDispatchedUnits // Number of Units Dispatched
-                                            <NbOfDispatchedUnitsByPlant // Number of Units
-                                                                        // Dispatched by plant
-                                             <ProfitByPlant
-                                              // Links
-                                              <Variable::Economy::Links // All links
-                                               >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
+                                       LOLP // Loss Of Load Probability
+                                       <NearPriceCap<
+                                         LOLP_CSR<AvailableDispatchGen<DispatchableGenMargin<
+                                           DtgMarginCsr // DTG MRG CSR
+                                           <Marge<MaxMrgCsr<NonProportionalCost<
+                                             NonProportionalCostByDispatchablePlant // Startup cost
+                                                                                    // + Fixed cost
+                                                                                    // per thermal
+                                                                                    // plant detail
+                                             <NbOfDispatchedUnits // Number of Units Dispatched
+                                              <NbOfDispatchedUnitsByPlant // Number of Units
+                                                                          // Dispatched by plant
+                                               <ProfitByPlant
+                                                // Links
+                                                <Variable::Economy::Links // All links
+                                                 >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
```

Comment:
i don't understand anything about this, i trust you :)

Reply:
It's basically a chained template, each function calling the next with the same name (convention).